src/components/@shared/FormFields/FilesInput/Info.tsx

```diff
@@ -3,6 +3,7 @@ import { prettySize } from './utils'
 import cleanupContentType from '@utils/cleanupContentType'
 import styles from './Info.module.css'
 import { FileMetadata } from '@oceanprotocol/lib'
+import Alert from '@shared/atoms/Alert/index'
```

Comment:
why add this import in this file?

Reply:
yeah, not need to. Issue already raised, we have fix some of them, but there's a still a couple to remove.